en/frames_multicopter/holybro_qav250_pixhawk4_mini.md

```diff
@@ -215,7 +215,7 @@ First update the firmware, airframe, and actuator mappings:
   
 - [Actuators](../config/actuators.md)
   - You should not need to update the vehicle geometry (as this is a preconfigured airframe).
-  - Assign actuator functions to outputs to match your wiring.
+  - Assign actuator functions to outputs to match your wiring. 
```

Comment:
@beniaminopozzan OK, so moved this to the line which makes sense. Are you sure this is correct? 

```suggestion
  - Assign actuator functions to outputs to match your wiring.
    - For the Pixhawk 4 Mini, and other controllers that do not have an [I/O board](../hardware/reference_design.md#mainio-function-breakdown), you will need to assign actuators to outputs on the `PWM AUX` tab.
    - The Pix32 v6 has an I/O board, so you can assign to either AUX or MAIN.    
```

Reply:
Thanks @hamishwillee !